templates/iam_policy/deploy_lambda_github_actions.json.tpl

```diff
@@ -33,6 +33,13 @@
         "arn:aws:lambda:${region}:${account_id}:function:tdr-signed-cookies-${environment}",
         "arn:aws:lambda:${region}:${account_id}:function:tdr-yara-av-${environment}",
         "arn:aws:lambda:${region}:${account_id}:function:tdr-reporting-${environment}",
+        "arn:aws:lambda:${region}:${account_id}:function:tdr-file-upload-data-${environment}",
```

Comment:
Could you put these in alphabetical order with the existing lambda function names.